gtsam/navigation/tests/testAttitudeFactor.cpp

```diff
@@ -47,11 +49,11 @@ TEST( Rot3AttitudeFactor, Constructor ) {
   Rot3 nRb;
   EXPECT(assert_equal((Vector) Z_2x1,factor.evaluateError(nRb),1e-5));
 
+  auto err_fn = [&factor](const Rot3& r){
+    return factor.evaluateError(r, OptionalNone);
```

Comment:
This seems messy just to be a bit more aligned to the Google style guide...

Reply:
I suspect this was just an oversight and that the old syntax could be used (and not a change driven by google style).  Although, personally, I actually find the lambda syntax easier to read than std::bind, but that's personal preference.

https://github.com/borglab/gtsam/pull/1392#discussion_r1083141328